authors/apps/articles/models.py

```diff
@@ -40,26 +43,26 @@ class Meta:
    
 
 class Comment(models.Model):
-   """
-   Model for comments
-   """
-   
-   body = models.CharField(max_length=200)
-   created_at = models.DateTimeField(auto_now_add=True)
-   updated_at = models.DateTimeField(auto_now=True)
-   author = models.ForeignKey(User, related_name='comments', on_delete=models.CASCADE)
-   article = models.ForeignKey(ArticlesModel, related_name='comments', on_delete=models.CASCADE)
-   parent = models.ForeignKey(
-                                'self',
-                                null=True,
-                                blank=True,
-                                related_name='threads',
-                                on_delete=models.CASCADE
-                            )
+    """
+    Model for comments
+    """
 
-   def __str__(self):
+    body = models.CharField(max_length=200)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User, related_name='comments', on_delete=models.CASCADE)
+    article = models.ForeignKey(ArticlesModel, related_name='comments', on_delete=models.CASCADE)
+    parent = models.ForeignKey(
```

Comment:
Could we have this as:
```python
 parent = models.ForeignKey(
    'self',
    null=True,
    blank=True,
    related_name='threads',
    on_delete=models.CASCADE
)
```
Just a minor formatting issue.

Reply:
Done

Reply:
Awesome 👍 . Just update the migration file to reflect the changes and it will be good to go